js/validation.js

```diff
@@ -0,0 +1,42 @@
+function validate()
+{
+	//document.body.appendChild(form);
+	
+	console.log("eho");
+	
+	let username = document.getElementById("username").value;
+	let usernamePattern = /^[a-zA-Z0-9_]{3,10}$/;
+	
+	let pass = document.getElementById("pass").value;
+	let passPattern = /^(?=.*[A-Z])(?=.*[a-z])(?=.*[0-9]).{6,}$/;
```

Comment:
http://webtech.w3c.fmi.uni-sofia.bg/w14labs/regex.html#2 "Now they have two problems." 
Would you say that this code is readable/easily understandable? Or easily modifiable by people who do not know regex? Also how error-prone is it to bugs?
Idea to consider: using multiple and simpler regex-es and applying "and" condition if all of them match